docs/dataset/formatting-your-datasets.md

```diff
@@ -23,26 +23,34 @@ dataset with the following columns:
 From this you can see that image `horse0000.png` has the ground_truth classification of `horse`,
 and has brightness and contrast data.
 
-An `id_field` is required in order to differentiate between datapoints.
-By default the `locator` or  `text` fields are used if present in your dataset, and other fields
-can be specified when importing via the Web App from the [:kolena-dataset-16: Datasets](https://app.kolena.io/redirect/datasets)
-page, or the SDK by using [`upload_dataset`](../reference/dataset/index.md#kolena.dataset.dataset.upload_dataset)
-function.
+When uploading a dataset to Kolena, it is important to be able to differentiate between each datapoint. This is
+accopmplished by configuring an `id_field` - an unique identifier for a datapoint. You can select any field that is
+unique across your data, for example a `uuid` or `article_id` for relevant data.
+
+Kolena handles the following `id_fields` as a special case:
+
+| Field Name  | Description                                                                                                                              |
+|-------------|------------------------------------------------------------------------------------------------------------------------------------------|
+| `locator`     | Url path to a file to be displayed, either a [cloud storage](../connecting-cloud-storage/index.md) url or a http url that serves a file. |
+| `text`        | Raw text input for text based models.                                                                                                    |
 
-A `locator` is a url path to a file that will be displayed on the platform and can either be a
-[cloud storage](../connecting-cloud-storage/index.md) url or a http url that serves a file.
 A locator needs to have correct extensions for the corresponding file type. For example an image should be in a format
 such as `.jpg` or `.png`, whereas locators for audio data should be in forms like `.mp3` or `.wav`.
 
-| Data Type   | Supported file formats                                                                |
-|-------------|---------------------------------------------------------------------------------------|
-| Image       | `jpg`, `jpeg`, `png`, `gif`, `bmp` and other web browser supported image types.       |
-| Audio       | `flac`, `mp3`, `wav`, `acc`, `ogg`, `ra` and other web browser supported audio types. |
-| Video       | `mov`, `mp4`, `mpeg`, `avi` and other web browser supported video types.              |
-| Document    | `txt` and `pdf` files.                                                                |
-| Point Cloud | `pcd` files.                                                                          |
+The table below shows the supported file formats for the supported data types:
```

Comment:
nit: can we drop one of the `supported` here?
I feel this can be replaces with a title for the table eg. `### Locator Support Matrix`

Reply:
Added a title for the table, good idea.